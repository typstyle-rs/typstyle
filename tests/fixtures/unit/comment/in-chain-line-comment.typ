#{
  ()
    // .rev()
    .rev()

  x
    // comment before method
    .method()

  a.b
    // comment before chain continuation
    .c.d

  obj
    // first comment
    .method1()
    // second comment  
    .method2()
}