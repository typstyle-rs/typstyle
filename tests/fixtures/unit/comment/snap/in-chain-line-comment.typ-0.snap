---
source: tests/src/unit.rs
input_file: tests/fixtures/unit/comment/in-chain-line-comment.typ
---
#{
  ()
    // .rev()
    .rev()

  x
    // comment before method
    .method()

  a
    .b
    // comment before chain continuation
    .c
    .d

  obj
    // first comment
    .method1()
    // second comment
    .method2()
}
